### PR TITLE
Fixes for Turkish translation files.

### PR DESCRIPTION
**Pull request by k-gun**

### CODE REVIEW

File: locale/tr/LC_MESSAGES/More/EAV/README.po [hunk 1]
```diff
@@ -17,7 +17,7 @@ msgstr "Entity-Attribute-Value (Varlık-Özellik-Değer)"
 
 #: ../../More/EAV/README.rst:4
 msgid "The Entity–attribute–value (EAV) pattern in order to implement EAV model with PHP."
-msgstr ""
+msgstr "-"
```
Comment (piotrgradzinski) [inline locale/tr/LC_MESSAGES/More/EAV/README.po]: Why "-"?

Comment (k-gun) [reply locale/tr/LC_MESSAGES/More/EAV/README.po]: 'Cos it does not add anything to translation. It sounds EAV is EAV.

Comment (piotrgradzinski) [reply locale/tr/LC_MESSAGES/More/EAV/README.po]: @k-gun first of all. If you put "-" in msgstr then this whole sentence "The Entity–attribute–value (EAV) pattern in order to implement EAV model with PHP." will be translated into "-" - this is not cool. Other thing is that this msgid is not a shortcut only but full sentence that should be translated as well.

File: locale/tr/LC_MESSAGES/More/Repository/README.po [hunk 2]
```diff
@@ -50,7 +50,7 @@ msgstr ""
 
 #: ../../More/Repository/README.rst:20
 msgid "Laravel Framework"
-msgstr ""
+msgstr "Aptal bir framework."
```
Comment (domnikl) [inline locale/tr/LC_MESSAGES/More/Repository/README.po]: why translate Laravel Framework? It's a name.

**Comment by domnikl** [comment]
Closing this as there was no more action on this and it seems that this adds little value and breaks translations as @piotrgradzinski mentioned.

File: locale/tr/LC_MESSAGES/More/Repository/README.po (hunk 2 above)
Comment (oguzdumanoglu) [reply locale/tr/LC_MESSAGES/More/Repository/README.po]: What! Translation says, 'It is a stupid framework.' ??

**Comment by elamblin** [comment]
Emmanuel Lamblin

 
 

    Le vendredi 22 décembre 2017 à 02:16:58 UTC+1, Kerem Güneş <notifications@github.com> a écrit :  
 
 
@k-gun commented on this pull request.

In locale/tr/LC_MESSAGES/More/EAV/README.po:
> @@ -17,7 +17,7 @@ msgstr "Entity-Attribute-Value (Varlık-Özellik-Değer)"
 
 #: ../../More/EAV/README.rst:4
 msgid "The Entity–attribute–value (EAV) pattern in order to implement EAV model with PHP."
-msgstr ""
+msgstr "-"

'Cos it does not add anything to translation. It sounds EAV is EAV.

**Pull request closed without merging**